modules/perception/production/conf/perception/lidar/velodyne128_segmentation_conf.pb.txt

```diff
@@ -1,5 +1,5 @@
 sensor_name: "velodyne128"
 enable_hdmap: true
-lidar_query_tf_offset: 0
+lidar_query_tf_offset: 200
```

Comment:
In the latest version of this PR, I restored this file and created a copy just for LGSVL Simulator, and changed this value there.

Without this change, the obstacle detection does not work well. It complains timestamp error. The reason may be because the timestamp of our sensors uses the simulation time, which is different to real time. With a large value of `lidar_query_tf_offset`, the obstacle detection is more stable.

Once you started to use our simulator, maybe you can tell me a better way to resolve this issue. We can change it further by then.